.codesandbox/ci.json

```diff
@@ -2,5 +2,5 @@
   "buildCommand": "build:codesandbox",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 6 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26928&commitId=df27ed0a7b1a23a4e79665dd994c72357e904b92&env=prod)

✅ There was 6 screenshots added, 0 screenshots removed, 1862 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|FluentProvider CustomStyleHooks.Button.chromium.png|0| Added|
|FluentProvider CustomStyleHooks.CompoundButton.chromium.png|0| Added|
|FluentProvider CustomStyleHooks.Default.chromium.png|0| Added|
|FluentProvider CustomStyleHooks.MenuButton.chromium.png|0| Added|
|FluentProvider CustomStyleHooks.SplitButton.chromium.png|0| Added|
|FluentProvider CustomStyleHooks.ToggleButton.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>